src/test/java/org/assertj/core/internal/char2darrays/Char2DArrays_assertHasSize_Test.java

```diff
@@ -24,7 +24,7 @@
 import org.junit.jupiter.api.Test;
 
 /**
- * Tests for <code>{@link Char2DArrays#assertHasSize(AssertionInfo, char[][], int, int)}</code>.
+ * Tests for <code>{@link Char2DArrays#assertHasDimensions(AssertionInfo, char[][], int, int)}</code>.
  *
```

Comment:
the class name should be renamed to `Char2DArrays_assertHasDimensions_Test`

Reply:
the same applies for other tests

Reply:
done